meetings/membership-criteria.md

```diff
@@ -0,0 +1,64 @@
+# Membership criteria
+
+When you apply for membership, we review your application based on the [membership participation types](membership-participation-types.md) defined. We ask for some basic information about you. 
+
+## Basic information
+
+First, we validate the personal information:
+
+- Reasonable first name, last name.
+- valid email address: We only validate that it is a valid email address format. Later, when an invitation is sent, that verifies that it's a real address.
```

Comment:
valid email address: We only validate that it is a valid email address format. Later, when an invitation is sent, that verifies that it's a real address.
should be
A valid email address: *We only validate that it is a valid email address format. Later, when an invitation is sent, that verifies that it's a real address.*